templates/.VERSION

```diff
@@ -1 +1 @@
-0.9.99.0915
\ No newline at end of file
+v0.9.99-212-g2c92e45
```

Comment:
The version file also could be `0.9.0+dev` so that `go get` will not report version error.